src/main/java/com/williamfiset/algorithms/datastructures/stack/impl/ArrayStack.java

```diff
@@ -0,0 +1,62 @@
+package com.williamfiset.algorithms.datastructures.stack.impl;
```

Comment:
To be consistent, lrave the package name as "com.williamfiset.algorithms.datastructures.stack" rather than "com.williamfiset.algorithms.datastructures.stack.impl"?

Reply:
fix it